modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -125,27 +130,28 @@ object FormInputEV {
 
     def render(p: Props[EV, A], s: State): VdomNode = {
 
-      val validationError: js.UndefOr[Label] =
-        s.errors
-          .map(e =>
-            Label(content = e.toList.mkString(","),
-                  clazz = p.errorClazz,
-                  pointing = p.errorPointing
-            )(^.position.absolute)
-          )
-          .orUndefined
+      def errorLabel(errors: NonEmptyChain[NonEmptyString]): js.UndefOr[ShorthandB[Label]] = {
+        val vdoms = errors.toList.map[VdomNode](_.value)
+        val list  = vdoms.head +: vdoms.tail.flatMap[VdomNode](e => List(<.br, <.br, e))
+        Label(
+          content = React.Fragment(list: _*),
+          clazz = p.errorClazz,
+          pointing = p.errorPointing
+        )(
+          ^.position.absolute
+        )
+      }
 
       val error: js.UndefOr[ShorthandB[Label]] = p.error
         .flatMap[ShorthandB[Label]] {
           (_: Any) match {
-            case b: Boolean => validationError.map(_.asInstanceOf[ShorthandB[Label]]).orElse(b)
-            case l: Label   =>
-              validationError
-                .map(vel => Label(content = React.Fragment(l, vel)).asInstanceOf[ShorthandB[Label]])
-                .orElse(l)
+            case b: Boolean => s.errors.map(errorLabel).getOrElse(b)
+            case e          => // We can't pattern match against NonEmptyString, but we know it is one.
+              val nes = e.asInstanceOf[NonEmptyString]
```

Comment:
Why can't we pattern match?

Reply:
> LGTM. There are, of course, conflicts with #148. Shouldn't be hard to sort out, though.

Sorry about that.

Reply:
> Why can't we pattern match?

Having `case e: NonEmptyString` results in a `non-variable type argument String in type pattern eu.timepit.refined.api.Refined[String,eu.timepit.refined.collection.NonEmpty] (the underlying of eu.timepit.refined.types.string.NonEmptyString) is unchecked since it is eliminated by erasure`